### PR TITLE
Moe sync

### PR DESCRIPTION
**Pull request by gk5885**

### CODE REVIEW

**Comment by ronshapiro** [comment]
LGTM

**Comment by cgruber** [comment]
There is at least one commit that I'd reword to account for the open source
project's meaning.  Notably the commit "pulling in javapoet" into our
corporate repository only partially applied to dagger, and the aspect that
applied was that we started depending on it.

**Comment by ronshapiro** [comment]
@cgruber That's been changed already, but no email was sent out about it.
